### PR TITLE
hero-large

### PR DESCRIPTION
**Pull request by tarawren**
https://trello.com/c/0zIyOXyy/19-section-hero-large

### CODE REVIEW

File: web/themes/fire/templates/components/hero-large/_hero-large.scss [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      left: 0;
+      right: 0;
+      width: 100%;
+      z-index: 1;
+    }
+
+    &--blue-green {
+      &:before {
+        content: '';
+        background: linear-gradient(210.85deg, #47AAAF 0%, #005C90 100%);
```
Comment (1forh) [inline web/themes/fire/templates/components/hero-large/_hero-large.scss]: I think this is the second time I'm seeing this. if it is, maybe it should be a mixin or something. thoughts?

Comment (tarawren) [reply web/themes/fire/templates/components/hero-large/_hero-large.scss]: Updated this and pulled it into `hero-medium` and `hero-small` as well.

File: web/themes/fire/templates/components/hero-large/_hero-large.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  @include media-breakpoint-down(sm) {
+    height: 330px;
+  }
+
+  &__media {
+    @include hero-before;
+    @include hero-edge-fallback;
+
+    &--blue-green {
+      @include hero-blue-green-gradient;
```
Comment (1forh) [inline web/themes/fire/templates/components/hero-large/_hero-large.scss]: I'm down to keep these as they are but you could totally do something like this if you wanted to. Seems a little cleaner to me.

```
@include hero-gradient('blue-green');
```

**Comment by tarawren** [comment]
@1forh updated

**Pull request merged**